nucliadb_vectors2/src/service/writer.rs

```diff
@@ -292,25 +292,29 @@ impl VectorWriter for VectorWriterService {
     }
 
     fn get_segment_ids(&self) -> NodeResult<Vec<String>> {
```

Comment:
I think the approach here and in `get_index_files` is good, but I think is now obvious that this functions should not be part of `VectorWriter` right?
Maybe we should create a `replication` module with those methods as functions whose input is the location of the index in the FS.
What do you think?

Reply:
This makes sense to me. The only reason it's not already done is to minimize changes in a single PR. But it might be worth doing it now.

Reply:
I think the change could fit in this PR, but no problem if you prefer to leave it for another PR!